common/test/keyboards/baseline/k_000___null_keyboard.kmn

```diff
@@ -1,8 +1,8 @@
 store(&NAME) '000 - null keyboard'
-c Description: Tests null keyboard 
+c Description: Tests null keyboard
 c keys: [K_A][RALT K_B][SHIFT K_C]
-c expected: aC
-c context: 
+c expected: C
+c context:
```

Comment:
`expected` should be testing the output that is expected from the keyboard -- not the context. So the `expected` output would still be `aC` here, because the `a` is never deleted. We need something else to verify the cached context (which should also verify markers); perhaps something like this?

```
c expected: aC
c expectedContext: C
c context:
```

Reply:
Yes we would need to add that. Did you see what I wrote above. 

> It could be a consideration to modify the text_store object to better
reflect the state of an application context however, this would mean
we would need two expected results one for the application and one
for the context.

Currently, the kmx.cpp believes expected === text_store == context;
Also note that these tests all assume non-context aware apps. Otherwise when the cache is invalidated on the next key stroke a Context aware app would just set the context again.

Reply:
Gotcha, I did miss that sorry. So we are on the same page. I think we should update the unit test accordingly, because of the ownership change of context.